met-api/src/met_api/schemas/comment.py

```diff
@@ -25,12 +22,20 @@ class Meta:  # pylint: disable=too-few-public-methods
     status_id = fields.Int(data_key='status_id')
     survey_id = fields.Int(data_key='survey_id')
     submission_id = fields.Int(data_key='submission_id')
-    comment_status = fields.Nested(CommentStatusSchema)
-    survey = fields.Pluck(SurveySchema, 'name')
-    question = fields.Method('get_comment_question')
+    status_id = fields.Method('get_comment_status_id')
+    reviewed_by = fields.Method('get_comment_reviewed_by')
+    label = fields.Method('get_comment_label')
+
+    def get_comment_status_id(self, obj):
+        """Get the associated status of the comment."""
+        return obj.submission.comment_status_id
+
+    def get_comment_reviewed_by(self, obj):
```

Comment:
you might be able to use fields.Pluck for these. Like fields.Pluck(SubmissionSchema, 'comment_status_id')

Reply:
Good call i did try that but it seems there's a circular reference problem, since Submission schema uses Comment schema I cannot add Submission Schema to comment schema, if that makes sense

Reply:
I see, makes sense